### DISCUSSION TITLE
About the project of vue version

### DISCUSSION THREAD

**Discussion by qian357891** [vuepress/core#1167] [General]
The next version of **vuepress** is 2.0.0-beta.53，the next version of **@vuepress/client** is 2.0.0-beta.53。
But the **next** version of **vue** is 3.2.36，the **latest** is 3.2.45。

If i code `pnpm i vue@next`（3.2.36）
 WARN  Issues with peer dependencies found
.
└─┬ vuepress 2.0.0-beta.53
  └─┬ vuepress-vite 2.0.0-beta.53
    └── ✕ unmet peer vue@^3.2.37: found 3.2.36

And when I code `pnpm doc:dev` it works.
But I code `pnpm doc:build` it will error:（both 3.2.36 and 3.2.37）
```
Error: usePageHead() is called without provider.
//...
```

And when I use `vue@3.2.45`（latest） it can be built。
I think it maybe a inadvertent problem.

**Comment by Mister-Hope** [comment]
Is any places in docs where we recommend you to use next tag for vue?

**Reply by qian357891** [reply]
I read the doc again。Sorry，It seems that I didn't read the documentation carefully.
